src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  background: #e5e5e5;
+}
+
+.wrapper {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  line-height: 60px;
+
+  text-decoration: none;
+  text-transform: uppercase;
+
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
better to move this color to some var and reuse it everywhere

Reply:
I didn't Sass the exam